pyproject.toml

```diff
@@ -50,10 +59,13 @@ select = [
   "RET",     # flake8-return
   "RSE",     # flake8-raise
   "RUF",     # Ruff-specific rules
+  # "S",     # flake8-bandit
```

Comment:
This [rule family](https://docs.astral.sh/ruff/rules/#flake8-bandit-s) provides some security related checks, it might be beneficial to enable it. But too many errors related to it, better to separate to another PR.